index.html

```diff
@@ -103,9 +103,46 @@ <h3 class="title-post">Multi-Post Stories</h3>
                                 <li class="tag-4">Ruby</li>
                             </ul>
                         </div>
-                        <button class="button-p-main" type="button">See Project</button>
+                        <button data-modal-target="#modal" class="button-p-main" type="button">See Project</button>
                     </div>
-
+                    <div class="modal" id="modal">
+                        <div class="modal-header">
+                            <div class="title">Keeping track of hundreds of components website</div>
+                            <button data-close-button class="close-button">&times;</button>
```

Comment:
- [ ] Your recent changes seem to have introduced a bug and the close button is not working anymore for the Pop-ups. Additionally, the technologies now seem to be overlapping each other as you can see below. Kindly fix these issues.
![image](https://user-images.githubusercontent.com/90556221/169623327-8f2be6a7-f885-4cd8-bab5-c7ea9375b9c5.png)

Reply:
Okey I will do these changes

Reply:
- [ ] Great job by implementing some of the requested changes from the previous reviewer, but as highlighted in the comment above, The close button doesn't work anymore. Once I click on it, nothing happens. Kindly, fix these issues before moving on to the next project.